docs/victory-line/ecology.md

```diff
@@ -117,7 +117,7 @@ VictoryLine also supports functional styles. Unlike other data components, style
 
 ### Animating
 
-VictoryLine animates with [VictoryAnimation](http://github.com/formidablelabs/victory-animation) as data changes when an `animate` prop is provided.
+VictoryLine animates with [VictoryAnimation][] as data changes when an `animate` prop is provided.
```

Comment:
Where will docs ultimately live?  We could link [here](https://github.com/FormidableLabs/victory-core/blob/master/docs/victory-animation/ecology.md) but not it that is likely to go away, or it is used to build a different docs page.